builtin/game/misc.lua

```diff
@@ -125,6 +125,18 @@ function core.get_player_radius_area(player_name, radius)
 end
 
 
+function core.is_valid_pos(pos)
+	if pos then
+		for _, v in pairs({"x", "y", "z"}) do
+			if not pos[v] or pos[v] < -32000 or pos[v] > 32000 then
```

Comment:
It'd probably be better if 32000 is replaced by the actual mapgen limit (defined by `mapgen_limit` setting).

Reply:
Updated, thanks